examples/seismic/model.py

```diff
@@ -37,6 +37,8 @@ def demo_model(preset, **kwargs):
                     filepath. Requires the ``opesci/data`` repository
                     to be available on your machine.
     """
+    space_order = kwargs.pop('space_order', 1)
```

Comment:
Would default to 2,  same as in `wavesolver`, I

Reply:
why?

Reply:
Because the default order for the seismic stencils is `2`,  o should be the model for consistency.

Reply:
oooh we're in ``seismic/model.py`` here, right...

Reply:
I can change it, I have the `dtype` fix too in `grid`